### PR TITLE
Suppressing some typing warnings

### PR DESCRIPTION
**Pull request by Diogo-Rossi**
WIP of #275

Replacing `pass` statement by Ellipsis object (`...`) which has the same result and suppress type warnings in return types.

### CODE REVIEW

**Comment by haiiliin** [comment]
Thanks for this PR!

File: src/abaqus/Amplitude/ActuatorAmplitude.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ def __init__(self, name: str, timeSpan: SymbolicConstant = STEP):
         RangeError
         """
         super().__init__()
-        pass
+        ...
```
Comment (haiiliin) [inline src/abaqus/Amplitude/ActuatorAmplitude.py:61]: I think we should remove the ellipse where there is already a statement, right?

**Review by haiiliin — COMMENTED** [review]
Looks good!

**Review by haiiliin — APPROVED** [review]

File: src/abaqus/Amplitude/ActuatorAmplitude.py (hunk 1 above)
Comment (Diogo-Rossi) [reply src/abaqus/Amplitude/ActuatorAmplitude.py]: Did it!
Now, there is no `pass` or Ellipsis where there is already a statement

**Review by haiiliin — APPROVED** [review]

File: src/abaqus/Amplitude/ActuatorAmplitude.py (hunk 1 above)
Comment (haiiliin) [reply src/abaqus/Amplitude/ActuatorAmplitude.py]: Merge it now

**Pull request merged**